LeezeIt/LeezeIt/Enums/Enums_StoryBoard.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Enums_StoryBoard.swift
+//  LeezeIt
+//
+//  Created by harris ali on 9/17/22.
+//
+
+import Foundation
+
+enum StoryBoardsID : String {
+    case LogInSignUp = "LogInSignUp"
```

Comment:
maybe we should name all of these like LogInSignUpStoryBoard

Reply:
resolved